src/documents/activities/StatesandContextsMapping.html.md.handlebars

```diff
@@ -0,0 +1,42 @@
+---
+title: States and Contexts Mapping
+layout: content
+category: Activities
+---
+
+This activity breaks down the User States and Contexts tool into modular pieces and actionable steps, and it is also a re-thinking of the original User States and Contexts tool.
+
+Here, states are reframed as functional requirements. Deficit language or disability classifications that are functionally ambiguous are avoided.
+
+Placing needs that are commonly met in the middle and needs that are not well met further out, demonstrates that edge needs are greater in number, and communicates the phenomena that by meeting edge needs, you also meet the needs in the middle.
```

Comment:
thanks @jhung, i've made those change and will also update the user states and contexts tool page.